### ISSUE TITLE
[feature] I would want a feature that automatically allows you to buy plans instead of purchasing them with coins :)

### ISSUE THREAD

**Issue by TheMinecraftExplorer** [ControlPanel-gg/dashboard#105]
# 🚀 Feature Request

### Is your proposal related to a problem?

*Provide a clear and concise description of what the problem is.
For example, "I can't change x and it makes me have to do manual work"*

I would want a system which allows me to purchase server plans automatically without using coins.

### Describe the solution you'd like

*Provide a clear and concise description of what you want to happen.
For Example, "Automate the changing of x so I don't have to do it manually"*

Basically you could make a feature like in whmcs, that you could just buy the plan instead of purchasing coins and then buying as its a hassle.
### Additional context

*Is there anything else you can add about the proposal?
You might want to link to related issues here, if you haven't already.*

If there is already a way of doing it, please let me know

**Comment by RamonRobben** [comment]
@TheMinecraftExplorer What makes this different than just buying coins? Either way its the same action. You buy something worth $5 and then you can create a server with whatever plan you want.

**Comment by AVMG20** [comment]
this is not how it works
